src/tools/io.lua

```diff
@@ -17,7 +17,7 @@ function _M.read_file(path)
 end
 
 function _M.write_to_file(path, value)
-  local file = io.open(path, "w")
+  local file, err = io.open(path, "w")
```

Comment:
I don't know, quick find. But I think it is only used by the CLI. It should. So it's fine.